src/Makefile.common.post

```diff
@@ -91,12 +91,18 @@ VS = $(subst .,_,$(notdir $(1)))_START
 	$(AS_LD) $(AS_LDFLAGS) -static $(if $($(call VS,$<)), \
 		--section-start .text=$($(call VS,$<))) -o $@.elf _$@
 	$(XOBJCOPY) -j .text -O binary $@.elf $@.bin
-	$(OBJCOPY) -I binary -O $(O_BDFARCH) \
-		--add-section .note.GNU-stack=/dev/null $@.bin $@_
 	nm -g -n _$@ | gawk '\
 		BEGIN {\
 		  print "// Warning: autogenerated";\
 	          print "";\
+	          print "asm(";\
+	          print "\".data\\n\"";\
+	          print "\".globl _binary_$(@:.o=)_o_bin_start\\n\"";\
+	          print "\"_binary_$(@:.o=)_o_bin_start:\\n\"";\
```

Comment:
Or actually, could you instead not add
the underscore for apple? In that case
the `asm()` additions in the .c files might
not be needed, everything can be solved
by this script?

Reply:
In that case this script needs to use something like:
`print "\".globl ASM_SYM(_binary_$(@:.o=)_o_bin_start)\\n\"";\`
where
```
#ifdef __APPLE__
#define ASM_SYM(x) _##x
#else
#define ASM_SYM(x) x
#endif
```
I can do it, I myself like asm("") better but if you insist I'll introduce the macro.

I'll need asm("") as well (or conditional underscores) in simx86 cpatch.c as well by the way.

Reply:
so basically there are 3 approaches:
1. `asm("")` always
2. `asm("")` only for __APPLE__
3. prefixing underscore only for __APPLE__ for symbols defined in asm sections